JDK8 is more strict than JDK7 when generating javadocs and this causes some
problems here:
https://www.redhat.com/archives/sclorg/2016-January/msg00010.html

diff -up swt-chart-0.9.0/pom.xml.test1 swt-chart-0.9.0/pom.xml
--- swt-chart-0.9.0/pom.xml.test1	2016-01-06 23:08:30.534089521 +0100
+++ swt-chart-0.9.0/pom.xml	2016-01-06 23:12:51.220726567 +0100
@@ -5,6 +5,9 @@
   <groupId>org.swtchart</groupId>
   <artifactId>swt-chart-0.9.0</artifactId>
   <version>0.0.1-SNAPSHOT</version>
+  <properties>
+    <additionalparam>-Xdoclint:none</additionalparam>
+  </properties>
   <packaging>pom</packaging>
   <modules>
     <module>org.swtchart</module>
